File: Lista_2/Exercicios_SALA-lista-2/EX2-Recuperacao-s/EX2-s.c
#include <stdio.h>

int main() {
    
    float n1, n2, n3;
    
    scanf("%f", &n1);
    scanf("%f", &n2);
    if ((n1 + n2) / 2 < 6){
        scanf("%f", &n3);
        if ((n1 + n2 + n3) / 3 < 5){
            printf("O aluno foi Reprovado com a média %.2f",(n1 + n2 + n3) / 3);
            }
        else{
            printf("O aluno foi Aprovado com a média %.2f",(n1 + n2 + n3) / 3);
        }
    }
    else{
        printf("O aluno foi Aprovado com a média %.2f",(n1 + n2 ) / 2);
    }
    
    return 0;
    
}

File: 01_IFSP/LP1/Lista_2/Exercicios_SALA-lista-2/EX2-Recuperacao-s/EX2-s.py
n1 = float(input())
n2 = float(input())
situacao = "aprovado"
n3 = 0
if (n1 + n2) / 2 < 6:
    n3 = float(input())
    if (n1 + n2 + n3) / 3 < 5:
        situacao = "reprovado"
print(f"O aluno foi {situacao} com a média {(n1 + n2 + n3) / 3}")

